src/standard_wrappers/nft_standards/dip_721_v2.ts

```diff
@@ -69,39 +85,48 @@ export default class ERC721 extends NFT {
 
     if ('Err' in metadataResult)
       throw new Error(
-        `${Object.keys(metadataResult.Err)[0]}: ${Object.values(metadataResult.Err)[0]
+        `${Object.keys(metadataResult.Err)[0]}: ${
+          Object.values(metadataResult.Err)[0]
         }`
       );
     const metadata = metadataResult.Ok;
-    const formatedMetadata = this.formatMetadata(metadata)
+    const formatedMetadata = this.formatMetadata(metadata);
+    const owner = metadata.owner[0] ? metadata.owner[0].toText() : 'aaaaa-aa';
```

Comment:
why fallback to icns canister?

Reply:
The owner field will always be in the contract with dip721v2, having it fallback (which should be never) fixes typescript complaining

Reply:
mmm what happens if the owner is not present for the UIs? I think we should never fail the request because of a missing field, just not sure if we should default to a canister Id cause it might bring some confusions.